tutorials/evoked/plot_evoked_measures.py

```diff
@@ -0,0 +1,153 @@
+"""
+.. _tut_erp:
+
+Event Related Potentials (ERPs) Measures
+==================================================
+
+.. contents:: Here we cover the specifics of extracting data from ERPs, namely:
+   :local:
+   :depth: 1
+
+"""
+
+import pandas as pd
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+
+###############################################################################
+# Create evoked from raw data
+
+# Read in raw data
+data_path = sample.data_path()
+raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+event_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw-eve.fif'
+raw = mne.io.read_raw_fif(raw_fname, preload=True)
+
+# Select only the the EEG channels
+raw.pick_types(meg=False, eeg=True, eog=True).load_data()
```

Comment:
I `.load_data()` is used here, then I guess `preload=False` was intended when reading.

Reply:
whoops...I missed that on my end. I've remove this from my recent version and I'm focusing on reading in the sample evoked `.fif` file